socorro/unittest/external/fs/test_tar_crashstore.py

```diff
@@ -1,85 +1,143 @@
-from nose.tools import eq_, ok_, assert_raises
-from mock import Mock, patch
+from nose.tools import ok_, eq_
+from mock import Mock
 from datetime import datetime
 
 import json
+import tarfile
+import gzip
```

Comment:
I know we don't have any strict guidelines on style, but common practice is to separate standard lib from third-party libs.